charts/consul/values.yaml

```diff
@@ -1859,6 +1859,85 @@ connectInject:
     # Note: This value has no effect if transparent proxy is disabled on the pod.
     defaultOverwriteProbes: true
 
+  # Configures consul-cni plugin for Consul Service mesh services
+  # Using this feature requires Consul 1.13.0-beta+
+  cni:
+    # If true, then all traffic redirection setup with use the consul-cni plugin
+    # @type: boolean
+    enabled: false
+
+    # Namespace to install cni plugin.
+    # @type: string
+    namespace: kube-system
```

Comment:
Why is it kube-system and not the same as the helm chart?

Reply:
Explanations are above.

Reply:
kube-system has been removed and the section is not required.